<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KernelAddressConvertTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>73</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>内核地址</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QTextEdit" name="address">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>999</width>
        <height>29</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>73</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>PML4索引</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="5">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="PML4_Index">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="PDPTE_Index">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="PDE_Index">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="PTE_Index">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="offset">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="5">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="PML4_Index_hex">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="PDPTE_Index_hex">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="PDE_Index_hex">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="PTE_Index_hex">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="offset_hex">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>29</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(248, 239, 255)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>73</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>PDPTE索引</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>73</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>PDE索引</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>73</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>PTE索引</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>73</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>offset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
